README.md

```diff
@@ -14,14 +14,25 @@ If you don't want to install from the web store for security reasons, you can in
 2. Visit [chrome://extensions](chrome://extensions) in your browser.
 3. Ensure that the Developer mode checkbox in the top right-hand corner is checked.
 4. Click `Load unpacked extension…` to pop up a file-selection dialog.
-5. Navigate to the directory in which the extension files live, and select it.
+5. Navigate to `src` in the extension directory, and select it.
 6. Visit any page on [gov.uk](https://gov.uk)
 
 Source: [Getting Started: Building a Chrome Extension](https://developer.chrome.com/extensions/getstarted#unpacked).
 
 ### Running the tests
 
-Visit [spec_runner.html](spec_runner.html) in your browser.
+Run the Jasmine test suite with:
+
+```
+$ bundle exec rake jasmine:ci
+```
+
+### Releasing the extension
+
+1. Update the version in `manifest.json`
+2. Run `rake build`
+3. Upload newly created package in `/build` to the [Chrome web store](https://chrome.google.com/webstore/developer/edit/dclfaikcemljbaoagjnedmlppnbiljen)
```

Comment:
Yes, it's on my personal account. I'm planning to move it to the GDS Google Apps account so that access can be shared.